src/newsblurapi.cpp

```diff
@@ -351,7 +351,7 @@ json_object* NewsBlurApi::query_api(const std::string& endpoint,
 	const std::string* postdata)
 {
 	std::string url = api_location + endpoint;
-	std::string data = utils::retrieve_url(url, cfg, "", postdata);
+	std::string data = utils::retrieve_url(url, cfg, "", postdata, "POST");
```

Comment:
This method, `query_api`, is a helper; other places in the class sometimes pass it `nullptr` to generate `GET`s. It looks like this should change the method depending on `postdata` being `nullptr` or not.